x-pack/plugins/maps/public/components/force_refresh_checkbox.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiFormRow, EuiSwitch, EuiSwitchEvent, EuiToolTip } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+interface Props {
+  applyForceRefresh: boolean;
+  setRespondToForceRefresh: (applyGlobalTime: boolean) => void;
+  isDisabled: boolean;
+}
+
+export function ForceRefreshCheckbox({
+  applyForceRefresh,
+  setRespondToForceRefresh,
+  isDisabled,
+}: Props) {
+  const onRespondRoForceRefreshChange = (event: EuiSwitchEvent) => {
+    setRespondToForceRefresh(event.target.checked);
+  };
+
+  const tooltipContent = isDisabled
+    ? i18n.translate('xpack.maps.filterEditor.applyForceRefreshDefaultHelp', {
+        defaultMessage: `Turn this flag off for when the underlying data is static, and does not need to be refreshed based on time.`,
+      })
+    : i18n.translate('xpack.maps.filterEditor.applyForceRefreshDisabledHelp', {
```

Comment:
How about "When enabled, auto refresh and manual refresh are applied to this layer".

cc @gchaps

Reply:
discussed offline. this is getting way convoluted. By disabling, I tried to remove a degree of freedom because turning on `apply force refresh`  when there is no time or query-based filtering, imho addresses a non-existent use-case. While this reduces the possible configurations, it makes documenting the usage of this flag harder.

Instead, will just keep this flag always switchable. It will keep documentation more simple, and internal code more simple as well.